### ISSUE TITLE
Issue about: IndexError: Invalid key: 0 is out of bounds for size 0

### ISSUE THREAD

**Issue by mvllwong** [hiyouga/ChatGLM-Efficient-Tuning#100]
This isuus is always regarding to self-genereated dataset lacking of certain column, like colonm name:input, output and so on. There isn't any default setting for those lacking  columns.

**Comment by mvllwong** [comment]
简单来说就是"instruct"和"output"不能出现空字符串

**Comment by hiyouga** [comment]
Instances with empty input or empty output will be neglected while data preprocessing. Make sure that the dataset contains at least one instance with non-empty input and output.

**Comment by mvllwong** [comment]
Yep, input here would be the same meaning as "instruction", or "prompt".
